tests/rustdoc/inline_cross/const-effect-param.rs

```diff
@@ -0,0 +1,29 @@
+// Regression test for issue #116629.
+// Check that we don't render host effect parameters & arguments.
+
+// aux-crate:const_effect_param=const-effect-param.rs
+// edition: 2021
+#![crate_name = "user"]
+
+// Don't render the host param on `load` and the host arg `host` passed to `Resource`.
```

Comment:
This one was already fixed by oli's PR but there was no cross-crate test for it yet, only a local crate one.